.ci/aws/Jenkinsfile

```diff
@@ -84,16 +81,12 @@ def run_test_orchestrator_once(run_name, build_tag, os, instance_type, instance_
 
     def cluster_name = get_cluster_name(build_tag, os, instance_type)
     def args = "--os ${os} --odcr ${odcr} --instance-type ${instance_type} --instance-count ${instance_count} --region ${region} --cluster-name ${cluster_name} ${addl_args} --junit-xml outputs/${cluster_name}.xml"
-    def ret = sh (
-                    script: ". venv/bin/activate; ./PortaFiducia/tests/test_orchestrator.py ${args}",
-                    returnStatus: true
-                  )
-    if (ret == 65)
-        unstable('Scripts exited with status 65')
-    else if (ret != 0)
-        build_ok = false
-    catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
-        sh "exit ${ret}"
+
+    try {
+        sh ". venv/bin/activate; ./PortaFiducia/tests/test_orchestrator.py ${args}"
```

Comment:
Why don't you have to check the return value of this shell call?

Reply:
it throws when there is a non-0 exit code.

Reply:
maybe we don't need the try/catch though